### ISSUE TITLE
Outline tab selections in orange

### ISSUE THREAD

**Issue by KatiRG** [CityofToronto/bdit_cot-vfh#5]
By default, the browser highlights them in a light blue which does not have enough contrast

![image](https://user-images.githubusercontent.com/1254764/69440129-37831300-0d16-11ea-9df9-d18083d34472.png)

**Comment by KatiRG** [comment]
See: https://formden.com/blog/input-color

Add to css:
```
.form-control:focus {
	border-color: #5cb85c;
	box-shadow: none;
	-webkit-box-shadow: none;
}
```

**Comment by KatiRG** [comment]
COT orange: #ec9f09

see VZ Map: https://www.toronto.ca/services-payments/streets-parking-transportation/road-safety/vision-zero/safety-measures-and-mapping/
